lib/modules/datasource/npm/get.ts

```diff
@@ -14,6 +14,8 @@ import { joinUrlParts } from '../../../util/url';
 import type { Release, ReleaseResult } from '../types';
 import type { CachedReleaseResult, NpmResponse } from './types';
 
+export const CACHE_REVISION = 2;
```

Comment:
why starting with `2` ? I would start with `1` as we didn't had it before
```suggestion
export const CACHE_REVISION = 1;
```